pkg/hub/spokecluster/manifests/spokecluster-role.yaml

```diff
@@ -3,5 +3,16 @@ kind: Role
 metadata:
   name: {{ .SpokeClusterName }}
```

Comment:
I think we're going to need names that distinguish
1. spoke-operator
2. spoke-registration
3. spoke-work

I'm having trouble keeping track and we don't have that many yet

Reply:
Agree.

I'm also not sure if this is the right place for these rules to live, but I think we can go with it for now.

Reply:
i will change the name of these roles so it would be easier to distinquish